tests/chainer_tests/links_tests/loss_tests/test_negative_sampling.py

```diff
@@ -88,88 +103,28 @@ def test_to_cpu(self):
         self.link.to_cpu()
         self.assertFalse(self.link.sampler.use_gpu)
 
-    @attr.gpu
-    def test_backward_cpu_gpu(self):
```

Comment:
Strictly speaking, this test is not redundant because there could be a situation that, for example, computation of gradients by both of numerical computation and backward propagation make the exactly same mistake. I do not think it happens so often.

I first thought it is OK to leave obsolete tests even a bit redundant. But the drawback of adding extra test cases is longer testing time. As we are suffering from long testing time, I think we need not to add tests that has little effect.